plugins/MauticFocusBundle/EventListener/ReportSubscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
```

Comment:
All new PHP files should declare strict types. 
```suggestion

declare(strict_types=1);

```